arrows/ocv/refine_detections_write_to_disk.cxx

```diff
@@ -148,21 +152,12 @@ ::refine( vital::image_container_sptr image_data,
   cv::Mat img = ocv::image_container::vital_to_ocv( image_data->get_image(), ocv::image_container::BGR_COLOR );
 
   // Get input filename if it's in the vital_metadata
-  std::string filename = "";
+  std::string filename;
```

Comment:
It's difficult to tell without the file handy but is there a case in which filename would remain unset and cause any harm above?

Reply:
Filename is only provided to the string formatter for creating the image file name. The default CTOR for string sets it to empty string, so assignment is redundant.